spec/rubykassa/payment_interface_spec.rb

```diff
@@ -13,16 +13,12 @@
     end
   end
 
-  it "should return correct base_url" do
-    @payment_interface.base_url.should == "http://test.robokassa.ru/Index.aspx"
-  end
-
   it "should return correct pay_url" do
-    @payment_interface.pay_url.should == "http://test.robokassa.ru/Index.aspx?MrchLogin=your_login&OutSum=1200&InvId=12&SignatureValue=bf0504363d89638669e057932857316c&shpfoo=bar"
+    @payment_interface.pay_url.should == "https://auth.robokassa.ru/Merchant/Index.aspx?MerchantLogin=your_login&OutSum=1200&InvId=12&IsTest=1&SignatureValue=bf0504363d89638669e057932857316c&shpfoo=bar"
   end
 
   it "should return correct pay_url when additional options passed" do
-    @payment_interface.pay_url({description: "desc", culture: "ru", email: "foo@bar.com", currency: ""}).should == "http://test.robokassa.ru/Index.aspx?MrchLogin=your_login&OutSum=1200&InvId=12&SignatureValue=bf0504363d89638669e057932857316c&shpfoo=bar&IncCurrLabel=&Desc=desc&Email=foo@bar.com&Culture=ru"
+    @payment_interface.pay_url({description: "desc", culture: "ru", email: "foo@bar.com", currency: ""}).should == "https://auth.robokassa.ru/Merchant/Index.aspx?MerchantLogin=your_login&OutSum=1200&InvId=12&IsTest=1&SignatureValue=bf0504363d89638669e057932857316c&shpfoo=bar&IncCurrLabel=&Desc=desc&Email=foo@bar.com&Culture=ru"
```

Comment:
Line is too long. [329/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.